container.go

```diff
@@ -435,7 +435,7 @@ func loadFifos(response *tasks.GetResponse) *cio.FIFOSet {
 		)
 		for _, fifo := range fifos {
 			if isFifo, _ := sys.IsFifo(fifo); isFifo {
-				if rerr := os.Remove(fifo); err == nil {
+				if rerr := os.Remove(fifo); err == nil && !os.IsNotExist(rerr) {
```

Comment:
Should we check earlier as well to avoid the extra call?

Reply:
@cpuguy83 IsFifo does do a similar check ( https://github.com/containerd/containerd/blob/master/sys/filesys.go#L26-L27 ), but we get the rug yanked below our feet between sys.IsFifo and os.Remove apparently